scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -165,6 +174,11 @@ function Initialize-IoTEdge {
         [ValidateNotNullOrEmpty()]
         [String] $DeviceTrustbundle,
 
+        # The external provisioning environment endpoint for the External provisioning mode.
+        [Parameter(Mandatory = $true, ParameterSetName = 'External')]
+        [ValidateNotNullOrEmpty()]
+        [String] $Endpoint,
```

Comment:
`$ExternalProvisioningEndpoint` perhaps? I know the manual- and dps-specific parameters don't mention `Manual` and `Dps` either, but `$Endpoint` seems too generic regardless.

Reply:
Sure, I can rename it.